BUILD.bazel

```diff
@@ -42,6 +42,8 @@ pandoc_toolchain(
     src = "README.md",
     from_format = "markdown",
     to_format = fmt,
+    # TODO(regisd) Infer extension from to_format
```

Comment:
Shall we fix this 'to do' before merging this PR?

Reply:
Done, this depended on #4